### PR TITLE
Experiment related refactor

### PR DESCRIPTION
**Pull request by shnela**

### CODE REVIEW

File: neptune/new/exceptions.py [hunk 1]
```diff
@@ -191,7 +238,26 @@ def __init__(
...
+        self.container_id = container_id
+        self.container_type = container_type
+        super().__init__(
+            "{} with ID {} not found. Could be deleted.".format(
+                container_type.value.capitalize(), container_id
+            )
+        )
+
+
+RunUUIDNotFound = ContainerUUIDNotFound
```
Comment (pkasprzyk) [inline neptune/new/exceptions.py:255]: 1. We don't raise it anywhere anymore, right?
2. Could you add a comment that it's only for backward compat etc?

File: neptune/new/exceptions.py [hunk 2]
```diff
@@ -191,7 +238,26 @@ def __init__(
...
+
+
+class ProjectNotFoundWithSuggestions(
```
Comment (pkasprzyk) [inline neptune/new/exceptions.py:258]: is this the only class extending `ExceptionWithProjectsWorkspacesListing`? if yes, I'd prefer to put its logic here (if nothing else, so we don't have to think/worry about multiple inheritance...).

File: neptune/new/exceptions.py (hunk 1 above)
Comment (shnela) [reply neptune/new/exceptions.py:256]: I've added a comment.

File: neptune/new/exceptions.py (hunk 2 above)
Comment (pkasprzyk) [reply neptune/new/exceptions.py:259]: ok, no discussion then

**Review by pkasprzyk — APPROVED** [review]

**Pull request merged**